edit.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+require_once 'functions/allFunctions.php';
+
+$db = connectToDB('pokemon-collection');
+$pokemonID = $_GET['id'];
+
+$userPokemon = fetchSingleUserPokemonData($db, $pokemonID);
+
+if(!$userPokemon){
+    header("location: index.php");
+    exit();
+}
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Edit Pokemon</title>
+</head>
+<body>
+<h1>Edit <?=($userPokemon['hasNickname']==1) ? $userPokemon['nickname'] . " the " : ""; echo $userPokemon['species'];?></h1>
+<?php echo displayPokemon($userPokemon,0); ?>
+<form method="post" action="editNickname.php">
+    <label>Give <?php echo $userPokemon['hasNickname']==1 ? $userPokemon['nickname'] : $userPokemon['species'];?> a new nickname? (Leave blank to clear nickname. Max chars: 12)
+        <input type="text" name="newNickname">
```

Comment:
You can give this input a maxlength attribute to prevent users from typing in more than 12 characters